examples/uwb_datasetgen/bagprocess_batch.py

```
@@ -0,0 +1,440 @@
+    rec_path = rec_path_list[0] if len(rec_path_list) > 0 else rec_path
+    src_name = src_path.split("/")[-1].split("_gt_odom")[0]
+    rec_name = rec_path.split("/")[-1].split("_gt_odom")[0]
+
+    rec_id = botids.index(rec_name)
+    src_id = botids.index(src_name)
+
+        
+    print("src_name:", src_name)
+    print("rec_name, rec_id:", rec_name, rec_id)
+    topic = "uwb_distance"
+    full_topic = "/"+src_name+"/"+rec_name+"/"+topic
```

Ruff (0.8.2)

Local variable `full_topic` is assigned to but never used

Remove assignment to unused variable `full_topic`

(F841)